README.md

```diff
@@ -36,6 +36,21 @@ Only required if the `SLACK_WEBHOOK_URL` environment variable is not set.
       with:
         webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
 
+#### `token` (optional)
+
+Only required if the `TOKEN` environment variable is not set.
+
+      with:
+        token: ${{ secrets.SLACK_TOKEN }}
+
+#### `channel_id` (optional)
```

Comment:
I don't understand why you need to create a new input field called `channel_id` when there is already `channel`. Please explain.